### PR TITLE
Allow deletion of multiple repos with faf repodel

### PR DESCRIPTION
**Pull request by michalfabik**
Make `faf repodel` accept one or more `REPO` arguments, add `--all` option.

Signed-off-by: Michal Fabik <mfabik@redhat.com>

### CODE REVIEW

**Comment by mgrabovsky** [comment]
Can we have some tests for this, too?

**Comment by mkutlak** [comment]
IMO pattern would be much more useful.

On retrace.fp.org we have repos like:
```
fedora-30-x86_64-one
fedora-30-x86_64-two
fedora-30-x86_64-three
fedora-31-x86_64-one
fedora-31-x86_64-two
fedora-31-x86_64-three
centos-7-x86_64-one
centos-7-x86_64-two
centos-7-x86_64-three
```

It would be much better to have `repodel --pattern="fedora-30*"` than `repodel --all`.

**Comment by michalfabik** [comment]
> IMO pattern would be much more useful.
> It would be much better to have `repodel --pattern="fedora-30*"` than `repodel --all`.  
  
Do we need the extra `--pattern` option? I rewrote it so the `REPO` argument(s) get treated as patterns outright.

**Comment by mkutlak** [comment]
> Do we need the extra `--pattern` option? I rewrote it so the `REPO` argument(s) get treated as patterns outright.

No, this is better :+1:

**Comment by michalfabik** [comment]
> Can we have some tests for this, too?

Some tests added. Not nearly as extensive as they could be, given the possibilities of wildcards.

File: src/pyfaf/actions/repodel.py [hunk 1]
```diff
@@ -25,24 +27,40 @@ class RepoDel(Action):
...
-        self.log_info("Removing repository '{0}'".format(cmdline.REPO))
+                self.log_info("Removing repository '{0}'".format(repo))
 
-        db.session.delete(repo)
-        db.session.flush()
+                db.session.delete(repo)
+
+            db.session.flush()
+        else:
+            self.log_error("No matching repositories found")
```
Comment (mkutlak) [inline src/pyfaf/actions/repodel.py]: Nitpick:
`log_info` (log_warn) instead of `log_error`?

Comment (michalfabik) [reply src/pyfaf/actions/repodel.py]: Fixed.

**Comment by mkutlak** [comment]
LGTM :+1:

**Pull request merged**